config/initializers/constant.rb

```diff
@@ -1 +1,2 @@
 APP_NAME = "Coeus Express"
+REDIRECT_BUYER_DOMAIN = "buyer.lvh.me:300"
```

Comment:
this is not required anymore right ?

Reply:
yeah but our issue is still unresolved for what we had added this.

Reply:
okay